docs/DriverParseableOutput.rst

```diff
@@ -76,13 +81,18 @@ key. An "outputs" object will have two fields, a "kind" describing the type of
 the output, and a "path" containing the path to the output. A "began" message
 will specify the command which was executed under the "command_executable" and
 "command_arguments" keys.
+To get the real process identifier for the process that started, get the value
+of "process.real_pid".
 
 Example::
 
    {
      "kind": "began",
      "name": "compile",
      "pid": 12345,
+     "process": {
+       "real_pid": 38732
```

Comment:
Isn't this impossible? From reading the documentation I inferred that the following must hold:

    pid == process.real_pid || pid < 0

If so, please update the example.

Reply:
Oh sure, you're right! I should also add that this will match the `pid` value if `pid` > 0 to the documentation!

Reply:
Excellent, thanks!

Reply:
Thanks for finding this! 👍🏻